calico_node/tests/st/bgp/test_ipip.py

```diff
@@ -140,56 +138,58 @@ def test_ipip_addr_assigned(self):
         with DockerHost('host', dind=False, start_calico=False) as host:
             # Set up first pool before Node is started, to ensure we get tunl IP on boot
             ipv4_pool = IPNetwork("10.0.1.0/24")
-            self.pool_action(host, "create", ipv4_pool, True)
+            self.pool_action(host, "create", ipv4_pool, ipip_mode=None)
             host.start_calico_node()
             self.assert_tunl_ip(host, ipv4_pool, expect=True)
 
             # Disable the IP Pool, and make sure the tunl IP is not from this IP pool anymore.
-            self.pool_action(host, "apply", ipv4_pool, True, disabled=True)
+            self.pool_action(host, "apply", ipv4_pool, ipip_mode="Always", disabled=True)
             self.assert_tunl_ip(host, ipv4_pool, expect=False)
 
             # Re-enable the IP pool and make sure the tunl IP is assigned from that IP pool again.
-            self.pool_action(host, "apply", ipv4_pool, True)
+            self.pool_action(host, "apply", ipv4_pool, ipip_mode="Always")
             self.assert_tunl_ip(host, ipv4_pool, expect=True)
 
             # Test that removing pool removes the tunl IP.
-            self.pool_action(host, "delete", ipv4_pool, True)
+            self.pool_action(host, "delete", ipv4_pool, ipip_mode=None)
             self.assert_tunl_ip(host, ipv4_pool, expect=False)
 
             # Test that re-adding the pool triggers the confd watch and we get an IP
-            self.pool_action(host, "create", ipv4_pool, True)
+            self.pool_action(host, "create", ipv4_pool, ipip_mode=None)
             self.assert_tunl_ip(host, ipv4_pool, expect=True)
 
             # Test that by adding another pool, then deleting the first,
             # we remove the original IP, and allocate a new one from the new pool
             new_ipv4_pool = IPNetwork("192.168.0.0/16")
-            self.pool_action(host, "create", new_ipv4_pool, True)
-            self.pool_action(host, "delete", ipv4_pool, True)
+            self.pool_action(host, "create", new_ipv4_pool, ipip_mode=None)
```

Comment:
wouldn't this create a disabled pool by default?

Reply:
I don't think so?  The `disabled` keyword argument defaults to `False` and its not specified here, so I think we should get the default, right?

Reply:
Oh yeah, you're right. Ignore that comment :)